packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,286 @@
+| Field button | Click area that opens the dropdown and displays the current selected option |
+| Dropdown     | A list of options to choose from displayed as an _open_ state               |
+| List option  | A selection the end-user can make, shown with other options in the dropdown |
+
+#### Label
+
+Selects should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases).
+
+Keep the label text as concise as possible, limited to a single line of text. Use sentence casing.
+
+In a form, use the label to indicate whether the select is optional or required for the end-user to interact with. If most of the components in the form are required, indicate which components are optional. If most of the components in the form are optional, indicate which components are required. Required or optional indication in the label text should always follow the pattern "[Label] (optional)" or "[Label] (required)".
+
+![ ](spec-images/select-2.png)
+
+#### Field Button
+
+Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that propts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.
+
+![ ](spec-images/select-4.png)
+
+Once the end-user has made a selection, they cannot clear the field button and get back to the empty state. Only use an empty state default in a select where there is no obvious default list option.
+
+#### Dropdown & List Options
+
+A list of options to select, displayed as an open state. Order list options alphabetically by default, unless another ordering will provide value to the end-user. Keep the list option text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-5.png)
+
+> NOTE: NEED VISUAL DESIGN GUIDANCE FOR ICON SIZES, WHAT CAN BE USED AS AN ICON, ETC.
+
+In addition to text, list options can include an icon to the left of the text, which also appears in the field button when selected. Icons should only be used when they provide more immediate communication to the user than the text. Do not include icons unless all list options have an icon.
+
+![ ](spec-images/select-7.png)
+
+### Related Components
+
+To allow end-users to enter a custom value, use a [**combobox**](/packages/nimble-components/src/combobox/specs/IxD.md).
+
+![ ](spec-images/select-alternative-combobox.png)
+
+> NOTE: SEE FUTURE CONSIDERATIONS FOR MULTI-SELECTION
+
+To allow end-users to multi-select list options, use a [**tag picker**](/packages/nimble-components/src/tag-picker/specs/IxD.md).
+
+![ ](spec-images/select-alternative-picker.png)
+
+To allow end-users to select less than three options, use a [**radio group**](/packages/nimble-components/src/radio-group/specs/IxD.md).
```

Comment:
nit: to avoid confusion vs multi-selection
```suggestion
To allow end-users to select from fewer than three options, use a [**radio group**](/packages/nimble-components/src/radio-group/specs/IxD.md).
```